testing/helpers/checkStyleHelper.js

```diff
@@ -0,0 +1,53 @@
+import devices from "core/devices";
+import browser from "core/utils/browser";
+
+function testInChromeOnDesktop(name, testCallback) {
+    if(devices.real().deviceType === "desktop" && browser.webkit) {
+        QUnit.testInActiveWindow.call(null, name, testCallback);
+    } else {
+        QUnit.skip.call(null, name, testCallback);
+    }
+}
+
+function checkTextOverflow(assert, element, expectedValue) {
+    let style = window.getComputedStyle(element.parentNode);
+    assert.equal(style.textOverflow, expectedValue, "textOverflow");
```

Comment:
we can return boolean value instead of using assert in the helpers

Reply:
Boolean result from this function means this code in test:

assert.ok(checkTextOverflow(style.textOverflow, expectedValue), "textOverflow");

With this code I will not see the 'expected' and 'actual' values in the test failure details.
This is why I want to use assert.equal with both these values.

However, I can change my functions to the 'getTextOverflow' approach and write my tests in this way:

assert.equal(CheckStyleHelper.getTextOverflow($template[0]), "clip", "textOverflow");

It keeps both values and allows to check styles of several elements in one test and provide different message for each assert . This approach looks better and I will change my code to implement it.

Reply:
I like it!

Reply:
Done